data/dev-rakeshshaw.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Rakesh Shaw",
+  "type": "personal",
+  "bio": "👋Upcoming **Dev**👋. Success requires persistent practice, guided by patience."
+  "avatar": "https://github.com/dev-rakeshshaw.png"
```

Comment:
Hi @dev-rakeshshaw There' no need to add the Avatar,  You can remove it.
 Thank you